DTO/PizzaViewModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace PizzaOrdering_Mvc.DTO
+{
+    public class PizzaViewModel
+    {
+        public Guid BaseId { get; set; }
+        public List<Guid> ToppingIds { get; set; } = new List<Guid>();
```

Comment:
Evaluate and mention why we need topping IDs in PizzaViewModel.

Reply:
@aitchcode to store multiple toppings for each pizza i've used List to store their ids